tests/Invoke-DbaQuery.Tests.ps1

```diff
@@ -153,7 +153,7 @@ SELECT @@servername as dbname
         }
         # Filter out the Verbose messages of the command Connect-DbaInstance
         $results = $results | Where-Object Out -NotMatch '\[Connect-DbaInstance\]'
-        $results.Length | Should -Be 7 # 6 'messages' plus the actual resultset
+        $results.Length | Should -BeIn 7, 8 # 6 'messages' plus the actual resultset plus maybe a message from Disconnect-DbaInstance if non-pooled connection
```

Comment:
dunno if this got added for a specific reason but Disconnect-DbaInstance output shouldn't be in the result pipeline ... why does it get there ?

Reply:
Adding the disconnect to the `end {}` _may_ help this

Reply:
The disconnect must be inside of the `foreach ($instance in $SqlInstance) {` loop, so moving to end is no option.
But I just had a look at line 155: We should include Disconnect-DbaInstance there to filter it out, don't we?

Reply:
I changed the test - please have a look.

Reply:
okay, now it's clear(er) . But given the fancy use of -Verbose specifically in Invoke-DbaQuery, I'd still mute fixedly anything coming out of Disconnect-DbaInstance in this. 
Theoretically all it should output should come from Invoke-DbaQuery only.
155 too has slipped in and shouldn't be there (messages from Connect-DbaInstance)